test/sequential/test-repl-persistent-history.js

```diff
@@ -94,6 +100,17 @@ const tests = [{
   expected: [prompt, replDisabled, prompt]
 },
 {
+  env: { NODE_REPL_HISTORY_FILE: emptyHistoryPath },
+  test: [UP],
+  expected: [prompt, convertMsg, prompt]
+},
+{
+  env: { NODE_REPL_HISTORY: emptyHistoryPath,
+         NODE_REPL_HISTORY_FILE: emptyHistoryPath },
```

Comment:
@zeusdeux The history is cleaned up and emptied by default in these tests, is it possible this one should be as follows?

```js
env: { NODE_REPL_HISTORY_FILE: defaultHistoryPath },
```